packages/libs/components/src/EntityDiagram/EntityDiagram.tsx

```diff
@@ -8,6 +8,7 @@ import {
 } from '@visx/hierarchy/lib/types';
 import { Line } from '@visx/shape';
 import { LinearGradient } from '@visx/gradient';
+import './diagram.css';
```

Comment:
This is the only non-formatting change in this file. After [fixing the CSS issues in `components`](https://github.com/VEuPathDB/web-monorepo/pull/122), we can now just let `EntityDiagram` set its own overflow property. I therefore [removed the overflow property being applied to the `.Entities` class](https://github.com/VEuPathDB/web-monorepo/pull/119/files?diff=split&w=1#diff-51e890452fe641482125521d5bae2e9d3a573a7c44c59f492c2a605ba347b992L162).